http://dev.mutt.org/trac/changeset/c837ab790d2e
http://dev.mutt.org/trac/ticket/1860

Abort prompt on ^C, as was the case prior to 12a6de725483


Index: curs_lib.c
===================================================================
--- curs_lib.c	(revision 5981:cff8e8ce4327)
+++ curs_lib.c	(revision 6010:c837ab790d2e)
@@ -99,5 +99,8 @@
 
   if (SigInt)
+  {
     mutt_query_exit ();
+    return err;
+  }
 
   if(ch == ERR)
